logzio-sender/src/test/java/io/logz/sender/InMemoryQueueTest.java

```diff
@@ -49,4 +60,35 @@ protected LogzioSender createLogzioSender(String token, String type, Integer dra
     protected void setZeroThresholdBuffer() {
         this.zeroThresholdBuffer = true;
     }
+
+    @Test
+    public void checkCrossCapacityInBytes() throws LogzioParameterErrorException {
```

Comment:
It's hard to see from the test written here that title you're testing "checkCrossCapacityInBytes" - we'll work on that